addOns/openapi/src/test/java/org/zaproxy/zap/extension/openapi/AbstractServerTest.java

```diff
@@ -59,11 +59,11 @@ protected static void checkPetStoreRequests(
         // Check all of the expected URLs have been accessed and with the right data
         assertTrue(accessedUrls.containsKey("POST " + baseUrl + "/pet"));
         assertEquals(
-                "{\"id\":10,\"category\":{\"id\":10,\"name\":\"John Doe\"},\"name\":\"John Doe\",\"photoUrls\":[\"John Doe\"],\"tags\":[{\"id\":10,\"name\":\"John Doe\"}],\"status\":\"available\"}",
+                "{\"id\":10,\"category\":{\"id\":10,\"name\":\"John Doe\"},\"name\":\"doggie\",\"photoUrls\":[\"John Doe\"],\"tags\":[{\"id\":10,\"name\":\"John Doe\"}],\"status\":\"available\"}",
```

Comment:
Better remove the example fields than change the tests, these should not test the example logic.